main.gms

```diff
@@ -633,11 +632,11 @@ parameter
 ***  (any value ge 0):  set costs multiplier to that value
 *'
 parameter
-  cm_1stgen_phaseout        "choose if 1st generation biofuels should phase out after 2030 (vm_deltaCap equals 0)"
+  cm_1stgen_phaseout        "scenario setting for phase-out of 1st generation biofuels"
 ;
   cm_1stgen_phaseout  = 0;         !! def = 0  !! regexp = 0|1
-*' *  (0): 1st generation biofuels after 2020 are fixed at upper limit of resource potential (maxprod)
-*' *  (1): no new capacities for 1st generation biofuel technologies may be built after 2030 -> phaseout until ~2060
+*' *  (0): no phase-out. Production of 1st generation biofuels after 2045 is bound from below by 90% of maximum resource potential ("maxprod")
+*' *  (1): phase-out. No new capacities for 1st generation biofuel technologies are built after 2030 (i.e. added capacity vm_deltaCap equals 0), in practice this means a slow phaseout of 1st generation biofuel due to lack of economic competitiveness. Bioenergy production is bound from below by 90% of maximum biomass resource potential in 2045
```

Comment:
I am not sure about that. @dklein-pik do you remember, what we discussed there? In the phase-out scenario (1) the description of the second part seems to contradict the phase-out. We can't have a phase-out and a lower bound (90% of potential) at the same time, right?

Reply:
yeah I am also a bit confused by this, but maybe Felix(?) @fschreyer implemented this bound because it conflicted with EDGE-T? https://github.com/remindmodel/remind/blob/develop/modules/30_biomass/magpie_40/bounds.gms#L24

Reply:
Well, I honestly do not remember the exact context in which those bounds were necessary. I think, I only [shifted them](https://github.com/remindmodel/remind/commit/42f09d2f7398cf9ac95cc3f1572f955aedeb80a9). Currently, it seems the vm_deltaCap is forced to zero from 2035 and vm_fuExtr is between 90-100% of the potential in 2045 for  both 0 and 1 values of the switch (and the lower bound continues after 2045 for the non-phaseout case). For the phase-out case, the 2045 90% lower bound does not make sense to me.

Reply:
Ok, I think, this switch needs some polishing content-wise. I just did two test runs (Baseline) to check, what it is doing and apparently the "phaseout = off" case actually leads to more bioenergy in the short term.
![pe_biomass_1st_generation](https://github.com/remindmodel/remind/assets/58845874/d4adf80f-b783-4b4c-8215-2e9dc18dffba)

Reply:
@merfort the last comment, you mean phaseout = On (green line) right?

I can leave the comment string as is, as it describes the current state of the code. When you do a patch, can tag me and I can take a look at the description again, if you want :)

Reply:
Ok, if `vm_deltaCap` is forced to be zero from 2035 on, that explains why the ramp up needs to be so quick in the phase-out scenario, when there is still that lower bound in 2045.

I will prepare a patch.